src/tenant/tests/test_views.py

```diff
@@ -136,6 +136,8 @@ def test_default(self):
         self.assertIn("Please Confirm Your E-mail Address", mail.outbox[0].subject)
         # expecting to see john.doe@example.com as recipient
         self.assertEqual(mail.outbox[0].to, ['john.doe@example.com'])
+        # expecting to see correct domain name in confirmation link
+        self.assertIn('http://default.localhost/accounts/confirm-email/', mail.outbox[0].body)
```

Comment:
This is a 404 page.  The test should verify the url is accessible and return 200 or redirect or something accessible.  That's been the problem and still is with this url.

Reply:
do you know how to run webserver from unittest to test if "http://default.localhost:8000" is running ?

Reply:
I've modified `get_email_confirmation_url` method (see latest commit) to use current (active) tenant object to build absolute uri for confirmation link, tests were updated too.